dts/arm/st/l1/stm32l151X8-a.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Please add matching `soc/arm/st_stm32/stm32l1/Kconfig.defconfig.stm32l151xba`, since both variants use different cmsis files (well, they are the same currently), and rely on different SoC define (STM32L151xBA vs STM32L151xB).

Reply:
ok, done.